ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -648,6 +648,7 @@ ethernet_interfaces:
     shutdown: < true | false >
     speed: < interface_speed | forced interface_speed | auto interface_speed >
     mtu: < mtu >
+    l2_mtu: < l2-mtu - only set if platform is matching 7050 >
```

Comment:
Change to `if defined this profile should only be used for platforms supporting the "l2 mtu" CLI` since it might also be applicable for 7358, 720XP etc.